pg/wal/heapdata.go

```diff
@@ -23,14 +23,16 @@ type HeapData interface {
 }
 
 // NewHeapData will interpret the heap data based on record type
-func NewHeapData(recordType RecordType, data []byte) HeapData {
+func NewHeapData(recordType RecordType, isInit bool, data []byte) []HeapData {
```

Comment:
Its a postgres internal thing but it means that a new page was initialized.  It doesn't really mean anything to us since we aren't updating heap files but it is one of two things that signify the 'offsets' array wont be there.